templates/account.php

```diff
@@ -163,26 +163,55 @@
         }
     }
 
-    $is_child_license    = ( is_object( $license ) && FS_Plugin_License::is_valid_id( $license->parent_license_id ) );
-    $bundle_subscription = null;
+    $has_bundle_license              = ( is_object( $license ) && FS_Plugin_License::is_valid_id( $license->parent_license_id ) );

```

Comment:
@fajardoleo for improved readability, I'd suggest to change it to:
```
$has_bundle_license = (
    is_object( $license ) &&
    FS_Plugin_License::is_valid_id( $license->parent_license_id )
);
```

Also, I'm not sure about the new name you gave to that variable as it doesn't check if there's a bundle license, but checks if the context license is a child of a bundle license.

Reply:
Actually, now that I reviewed the code below, I better understand the decision about the variable name. Therefore, a better way to write that code from a semantical point of view is as follows:
```
$has_bundle_license = false;

if ( is_object( $license ) &&
     FS_Plugin_License::is_valid_id( $license->parent_license_id )
) {
    // Context license has a parent license, therefore, the account has a bundle license.
    $has_bundle_license = true;
}
```